examples/client/tui/actions.nim

```diff
@@ -411,3 +411,22 @@ proc action*(self: Tui, event: UserMessageEvent) {.async, gcsafe, nimcall.} =
 
     debug "TUI received user message", message, timestamp, username
     self.printMessage(message, timestamp, username, topic)
+
+# CallEvent --------------------------------------------------------------------
```

Comment:
Not a big deal, but it's nice if the name in the comment matches the type name.

Actually, at this point it probably makes to create an `acitons/` dir and start splitting actions into separate modules. That can be done in a separate PR, of course; I'm just thinking aloud.